scalability_and_performance/ztp-zero-touch-provisioning.adoc

```diff
@@ -8,14 +8,21 @@ toc::[]
 
-Zero touch provisioning addresses the problem of provisioning new telco sites from the moment there is network connectivity to the site and until the CNF workload is deployed and running on the site nodes.
+Zero touch provisioning (ZTP) addresses the problem of provisioning new telco sites from the moment there is network connectivity to the site and until the CNF workload is deployed and running on the site nodes.
 
-ZTP eliminates the need for service providers to manually configure bare metal equipment at remote sites. Manual intervention is eliminated.  Declarative frameworks of desired end states are pushed out into the environment. Template or overlay configurations install {product-title} features that are required for CNF workloads and an end-to-end functional test suite to verify CNF related features. All Kustomize configs are entirely declarative in nature.
+ZTP eliminates the need for service providers to manually configure bare-metal equipment at remote sites. Manual intervention is eliminated. Declarative frameworks of desired end states are pushed out into the environment. Template or overlay configurations install {product-title} features that are required for CNF workloads and an end-to-end functional test suite to verify CNF related features. All Kustomize configs are entirely declarative in nature.
 
-Customers start by creating and refining ISO images of the node configuration they want. A unique image for each node is created based on the configuration provided. The images are then used to repeatedly provision vast numbers of nodes efficiently and quickly, allowing the customer to keep up with the needs from the field for far edge nodes.
+Customers start by creating and refining ISO images of the node configuration they want. A unique image for each node is created based on the configuration provided. The images are then used to repeatedly provision large numbers of nodes efficiently and quickly, allowing the customer to keep up with the needs from the field for far edge nodes.
 
-Service providers are deploying a more distributed mobile network architecture allowed by the modular functional framework defined for 5G and moving from appliance-based radio access networks (RAN) to open cloud RAN architecture to gain flexibility and agility in delivering services to their customers.
+Service providers are deploying a more distributed mobile network architecture allowed by the modular functional framework defined for 5G. This allows service providers to move from appliance-based radio access networks (RAN) to open cloud RAN architecture, gaining flexibility and agility in delivering services to end users.
 
+//planning and prereqs
+
+include::modules/ztp-site-planning-for-du-deployments.adoc[leveloffset=+1]
+
+include::modules/ztp-du-host-bios-requirements.adoc[leveloffset=+1]
+
+include::modules/ztp-du-host-networking-requirements.adoc[leveloffset=+1]
```

Comment:
Please double-check the render and make sure you're happy with the placement of this module in the assembly. Scanning the output, I don't know what you're supposed to do with these parameters. I don't see where you tell the user to make the CRs that have these fields or which CRs they're in, but I see some of them mentioned in the "About ZTP and distributed units on single nodes" section.

Reply:
I take your point. The additional content is going to come after GA with the "deploying DUs" user story.